filecoin-proofs/src/constants.rs

```diff
@@ -63,9 +85,15 @@ lazy_static! {
     pub static ref WINDOW_POST_SECTOR_COUNT: RwLock<HashMap<u64, usize>> = RwLock::new(
         [
             (SECTOR_SIZE_2_KIB, 2),
+            (SECTOR_SIZE_4_KIB, 2),
+            (SECTOR_SIZE_16_KIB, 2),
+            (SECTOR_SIZE_32_KIB, 2),
             (SECTOR_SIZE_8_MIB, 2),
+            (SECTOR_SIZE_16_MIB, 2),
             (SECTOR_SIZE_512_MIB, 2),
-            (SECTOR_SIZE_32_GIB, 2349) // this gives 133,977,564 constraints, fitting just in a single partition
+            (SECTOR_SIZE_1_GIB, 2349), // this gives 133,977,564 constraints, fitting just in a single partition
+            (SECTOR_SIZE_32_GIB, 2),
+            (SECTOR_SIZE_64_GIB, 2349), // FIXME
```

Comment:
BLOCKING: We need to finalize this number to resolve the FIXME. (cc: @dignifiedquire)

Reply:
fixed